### PR TITLE
chngefeed (ticdc): move resolvedTs into globalBarrierTs when redo log is enabled.

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: ref #9058

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/processor/processor.go [hunk 1]
```diff
@@ -219,7 +219,7 @@ func (p *processor) IsAddTableSpanFinished(span tablepb.Span, isPrepare bool) bo
 		return false
 	}
 
-	globalResolvedTs := p.changefeed.Status.ResolvedTs
+	// globalResolvedTs := p.changefeed.Status.ResolvedTs
```
Comment (sdojjy) [inline cdc/processor/processor.go:222]: it's better to remove this line

**Review by sdojjy — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

**Pull request merged**